Code/GraphMol/Wrap/rough_test.py

```diff
@@ -5908,6 +5908,21 @@ def testRemoveHsParams(self):
     m = Chem.RemoveHs(m, ps)
     self.assertEqual(m.GetNumAtoms(), 1)
 
+    m = Chem.MolFromSmiles('c1c(C([2H])([2H])F)cccc1', smips)
+    ps = Chem.RemoveHsParameters()
+    m_noh = Chem.RemoveHs(m, ps)
+    self.assertEqual(m_noh.GetNumAtoms(), m.GetNumAtoms())
+    ps.removeAndTrackIsotopes = True
+    m_noh = Chem.RemoveHs(m, ps)
+    self.assertEqual(m_noh.GetNumAtoms(), m.GetNumAtoms() - 2)
+    self.assertTrue(m_noh.GetAtomWithIdx(2).HasProp("_isotopicHs"))
+    self.assertEqual(tuple(map(int,
+        m_noh.GetAtomWithIdx(2).GetProp("_isotopicHs").split())), (2,2))
+    m_h = Chem.AddHs(m_noh)
+    self.assertFalse(m_h.GetAtomWithIdx(2).HasProp("_isotopicHs"))
+    self.assertEqual(sum([1 for nbr in m_h.GetAtomWithIdx(2).GetNeighbors()
```

Comment:
a philosophical comment, not really suggesting a change:
Though this construction directly tests what you're trying to test, it's reasonably non-trivial to read and figure out what it's doing. In order to lessen the cognitive load on the person reading through the tests, I'd probably just use MolToSmiles() to test that the right thing has happened here.

Reply:
Duly noted. I have done much worse than that, though, as you point out later on : )